From: Gunnar Wolf <gwolf@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2009-12-03
Description: Moves the default logging location to its own directory

Index: cherokee/cherokee.conf.sample.pre
===================================================================
--- cherokee/cherokee.conf.sample.pre	2011-06-10 09:40:18.000000000 -0500
+++ cherokee/cherokee.conf.sample.pre	2011-06-10 09:41:46.000000000 -0500
@@ -27,11 +27,11 @@
 
 vserver!1!logger = combined
 vserver!1!logger!access!type = file
-vserver!1!logger!access!filename = %localstatedir%/log/cherokee.access
+vserver!1!logger!access!filename = %localstatedir%/log/cherokee/cherokee.access
 vserver!1!logger!access!buffsize = 16384
 
 vserver!1!error_writer!type = file
-vserver!1!error_writer!filename = %localstatedir%/log/cherokee.error
+vserver!1!error_writer!filename = %localstatedir%/log/cherokee/cherokee.error
 
 vserver!1!rule!1!match = default
 vserver!1!rule!1!handler = common
